jvm/jfr-annotations/src/main/java/org/quickperf/jvm/jmcrule/JmcRuleCountMeasureExtractor.java

```diff
@@ -13,46 +13,61 @@
 package org.quickperf.jvm.jmcrule;
 
 import org.openjdk.jmc.common.item.IItemCollection;
+import org.openjdk.jmc.common.unit.IQuantity;
 import org.openjdk.jmc.common.util.IPreferenceValueProvider;
-import org.openjdk.jmc.flightrecorder.rules.IRule;
-import org.openjdk.jmc.flightrecorder.rules.Result;
-import org.openjdk.jmc.flightrecorder.rules.RuleRegistry;
-import org.openjdk.jmc.flightrecorder.rules.Severity;
+import org.openjdk.jmc.flightrecorder.rules.*;
+import org.openjdk.jmc.flightrecorder.rules.jdk.cpu.HighJvmCpuRule;
+import org.openjdk.jmc.flightrecorder.rules.jdk.memory.GarbageCollectionInfoRule;
+import org.openjdk.jmc.flightrecorder.rules.jdk.memory.GcStallRule;
+import org.openjdk.jmc.flightrecorder.rules.jdk.memory.HeapInspectionRule;
+import org.openjdk.jmc.flightrecorder.rules.jdk.memory.SystemGcRule;
 import org.quickperf.ExtractablePerformanceMeasure;
 import org.quickperf.jvm.jfr.JfrRecording;
 import org.quickperf.unit.Count;
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.RunnableFuture;
+import java.util.stream.Collectors;
 
 public class JmcRuleCountMeasureExtractor implements ExtractablePerformanceMeasure<JfrRecording, JmcRulesMeasure> {
 
     public static final JmcRuleCountMeasureExtractor INSTANCE = new JmcRuleCountMeasureExtractor();
-
-    private JmcRuleCountMeasureExtractor() {}
+    private static final List<Class<? extends IRule>> IGNORED_RULES = Arrays.asList(
```

Comment:
Why are these rules skipped?

Reply:
They are causing some NullPointerException-s inside the third party library when they are being evaluated.

Reply:
Could you please add a comment explaining this?

Reply:
Sure

Reply:
Done